### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by parvez-rashid**
Created avatar and badge components.

Live preview: [https://l5xr1.csb.app/](url)

### CODE REVIEW

File: avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+:root {
+  --primary-color: #bc3f4a;
+  --white: #f3f4f6;
```
Comment (manish-GH) [inline avatar/avatar.css:3]: instead of --white, you could give a more descriptive name like --text-color-white

File: badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
+@import "/avatar/avatar.css";
+
+:root {
+  --green: #22c55e;
```
Comment (manish-GH) [inline badge/badge.css:4]: Same case here, use a descriptive name

**Review by manish-GH — APPROVED** [review]
The code looks good. Next time do attach the live version of the app too

File: avatar/avatar.css (hunk 1 above)
Comment (parvez-rashid) [reply avatar/avatar.css:3]: Sure, will take care of that in the subsequent refactors.

**Comment by parvez-rashid** [comment]
> The code looks good. Next time do attach the live version of the app too

Sure, will keep this in mind.

**Comment by Naman-Saxena1** [comment]
Code looks fine to me 👍

**Comment by rajaranjanmr** [comment]
code looks fine to me. You can merge

**Review by rajaranjanmr — APPROVED** [review]

**Pull request merged**